.travis.yml

```diff
@@ -0,0 +1,36 @@
+os: linux
+dist: bionic
+language: bash
+services: docker
+
+branches:
+  only:
+  - master
+
+env:
+  - VERSION=4.5
+  - VERSION=4.6
+  - VERSION=4.7
+
+jobs:
+  include:
+    - stage: Build x86_64 image
+      arch: amd64
+      script:
+        - docker build --build-arg RELEASE_VER=$VERSION -t openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-$(uname -m) -f images/Dockerfile .
+        - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+        - docker push openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-$(uname -m)
+    - stage: Build ppc64le image
+      arch: ppc64le
+      script:
+        - docker build --build-arg RELEASE_VER=$VERSION -t openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-$(uname -m) -f images/Dockerfile .
+        - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+        - docker push openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-$(uname -m) 
+    - stage: Build multi-arch image
+      script: 
+        - export DOCKER_CLI_EXPERIMENTAL=enabled
+        - docker manifest create openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION --amend openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-x86_64 --amend openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-ppc64le
+        - docker manifest create openshift-install-powervs:ocp$VERSION --amend openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-x86_64 --amend openshift-install-powervs:$TRAVIS_TAG-ocp$VERSION-ppc64le
```

Comment:
Can we use docker tag since it is the same image, instead of creating manifest again?

Reply:
docker tag cannot be used on the manifest list since 'docker manifest create' creates a list and not a docker image.